packages/main/src/ColorPaletteItem.ts

```diff
@@ -52,6 +52,19 @@ class ColorPaletteItem extends UI5Element implements ITabbable {
 	@property({ validator: CSSColor })
 	value!: string;
 
+	/**
+	 * Defines if the component is selected.
+	 * <br><br>
+	 * Only one item can be selected per <code>ui5-color-palette</code>.
```

Comment:
This statement is not correct: not "can", but "must", also this does not depend on ColorPaletteItem itself. You can add it as note.